internal/kafka/test/mocks/kasfleetshardsync/kas-fleetshard-sync.go

```diff
@@ -279,18 +280,20 @@ func SampleDataPlaneclusterStatusRequestWithAvailableCapacity() *private.DataPla
 			},
 		},
 		Remaining: private.DataPlaneClusterUpdateStatusRequestTotal{
-			Connections:                   &[]int32{1000000}[0], // TODO set the values taking the scale-up value if possible or a deterministic way to know we'll pass it
-			Partitions:                    &[]int32{1000000}[0],
-			IngressEgressThroughputPerSec: &[]string{"test"}[0],
-			DataRetentionSize:             &[]string{"test"}[0],
+			Connections:             &[]int32{1000000}[0], // TODO set the values taking the scale-up value if possible or a deterministic way to know we'll pass it
+			Partitions:              &[]int32{1000000}[0],
+			IngressThroughputPerSec: &[]string{"test"}[0],
```

Comment:
The CI is complaining because this change (along side other changes in the test folders) will need to still reference `IngressEgress...` since we won't be changing the Status structs.

Reply:
Yep, fixed

Reply:
Awesome, thanks.